apps/api-extractor/src/api/mixins/ApiResultTypeMixin.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.s
+
+import { ApiItem, IApiItemJson, IApiItemConstructor, IApiItemOptions } from '../model/ApiItem';
+
+/** @public */
+export interface IApiResultTypeMixinOptions extends IApiItemOptions {
+  resultTypeSignature: string;
+}
+
+export interface IApiResultTypeJson extends IApiItemJson {
+  resultTypeSignature: string;
+}
+
+const _resultTypeSignature: unique symbol = Symbol('ApiResultTypeMixin._resultTypeSignature');
+
+/** @public */
+// tslint:disable-next-line:interface-name
+export interface ApiResultTypeMixin extends ApiItem {
+  readonly resultTypeSignature: string;
+  serializeInto(jsonObject: Partial<IApiItemJson>): void;
+}
+
+/** @public */
+export function ApiResultTypeMixin<TBaseClass extends IApiItemConstructor>(baseClass: TBaseClass):
```

Comment:
>ApiResultTypeMixin [](start = 16, length = 18)

The word "Result" doesn't really make sense for parameters. Maybe `ApiTypedMixin`?